test/verify/check-storage-btrfs

```diff
@@ -0,0 +1,560 @@
+        subvol_name = os.path.basename(subvol)
+
+        m.execute(f"mkfs.btrfs -L {label} -d raid1 {disk1} {disk2}")
+        self.login_and_go("/storage")
+
+        # creation of btrfs partition can take a while on TF.
+        with b.wait_timeout(30):
+            b.wait_visible(self.card_row("Storage", name=label))
+
+        b.wait_in_text(self.card_row("Storage", name=os.path.basename(disk1)), "btrfs device")
+        b.wait_in_text(self.card_row("Storage", name=os.path.basename(disk2)), "btrfs device")
+
+        # mount /
+        self.click_dropdown(self.card_row("Storage", name=label) + " + tr", "Mount")
+        self.dialog({"mount_point": mount_point})
+        b.wait_visible(self.card_row("Storage", location=mount_point))
+
+        # create subvolume
+        m.execute(f"""
+            btrfs subvolume create {subvol}
+            btrfs subvolume create {subvol2}
+        """)
+        b.wait_visible(self.card_row("Storage", name=os.path.basename(subvol)))
+
+        self.click_dropdown(self.card_row("Storage", name=os.path.basename(subvol)), "Mount")
+        self.dialog({"mount_point": subvol_mount_point})
+        b.wait_visible(self.card_row("Storage", location=subvol_mount_point))
+
+        # devices overview
+        self.click_card_row("Storage", name=label)
+        b.wait_visible(self.card_row("btrfs volume", name=disk1))
+        b.wait_visible(self.card_row("btrfs volume", name=disk2))
+
+        # Allocate some space to see if the space is spread evenly
+        # m.execute(f"fallocate -l 50M {mount_point}/bigfile")
+
+        # unmount via main page
+        b.go("#/")
+        b.wait_visible(self.card("Storage"))
+
+        self.click_dropdown(self.card_row("Storage", location=subvol_mount_point), "Unmount")
+        self.confirm()
+        b.wait_visible(self.card_row("Storage", location=f"{subvol_mount_point} (not mounted)"))
+
+        self.click_dropdown(self.card_row("Storage", name=os.path.basename(subvol)), "Mount")
+        self.dialog({"mount_point": subvol_mount_point})
+        b.wait_visible(self.card_row("Storage", location=subvol_mount_point))
+
+        mount_options = m.execute(f"findmnt --fstab -n -o OPTIONS {subvol_mount_point}").strip()
+        self.assertIn(f"subvol={subvol_name}", mount_options)
+        self.assertEqual(mount_options.count(subvol_name), 1)
+
+        self.click_dropdown(self.card_row("Storage", name=os.path.basename(subvol2)), "Mount")
+        self.dialog_wait_open()
+        self.dialog_set_val("mount_point", subvol_mount_point)
+        self.dialog_apply()
+        self.dialog_wait_error("mount_point", f"Mount point is already used for subvolume {os.path.basename(subvol)}")
+        self.dialog_cancel()
+
+        self.click_dropdown(self.card_row("Storage", location=subvol_mount_point), "Unmount")
+        self.confirm()
+        b.wait_visible(self.card_row("Storage", location=f"{subvol_mount_point} (not mounted)"))
+
+    def testSubvolumeMount(self):
```

Comment:
Let's call this testSubvolumeMountingHelp, because that's what it is about.

Reply:
Hmm, isn't this actually covered by testMountingHelp?

Reply:
testMountingHelp is copied from the other storage tests, I want to unify that at some point in the future. This specific test was added during development for a specific issue where subvol= was added multiple times but it indeed largely overlaps so let's remove it.